Svc/Deframer/docs/sdd.md

```diff
@@ -129,7 +129,13 @@ See <a href="#dpi-impl">below</a> for a description of how `Deframer` implements
 
 Here is a class diagram for `Deframer`:
 
-![classdiagram](./img/class_diagram_deframer.png)
+```mermaid
+classDiagram
+    ObjBase <|-- PassiveComponentBase
+    PassiveComponentBase <|-- DeframerComponentBase
+    DeframerComponentBase <|-- DeframerComponentImpl
```

Comment:
Let's remove the `ComponentImpl` suffix here to bring the diagram in line with the newest version of the implementation. Same on the next line.

Reply:
Fixed